js/main.js

```diff
@@ -0,0 +1,187 @@
+  },
+  {
+    image: './images/speaker_05.png',
+    name: 'Lila Tretikov',
+    profession: 'Executive Director of the Wikimedia Foundation',
+    description: 'Lila Tretikov is the Exectuvie Director of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely avaiable un 290 languages and used by nearly half a billion people around the world every month',
+  },
+  {
+    image: './images/speaker_06.png',
+    name: 'Ryan Merkley',
+    profession: 'CEO of Creative Commons, ex COO of the Mozilla Foundation',
+    description: 'Ryan had been leading open-source projects as the Mozilla Foundation such as the open source movement.',
+  },
+];
+
+const printSpeakers = () => {
+  const speakersSection = document.getElementById('featuredSpeakers');
+  speakers.forEach((speaker) => {
+    const ul = document.createElement('ul');
+    ul.classList.add('speakers__list__container');
+    ul.innerHTML = `
+        <li class="featured__speakers__item">
+            <div class="speaker__description__container">
+                <figure>
+                    <img src="${speaker.image}" alt="" class="featured__speaker__image">
+                </figure>
+                <div class="featured__speaker__nameAndProfession">
+                    <h3 class="featured__speaker__name">${speaker.name}</h3>
+                    <h4 class="featured__speaker__profession">${speaker.profession}</h4>
+                    <hr>
+                    <p class="featured__speaker__description">${speaker.description}</p>
+                </div>
+            </div>
+        </li>
+        `;
+    speakersSection.appendChild(ul);
+  });
+};
+printSpeakers();
+
+const showMore = () => {
+  if (window.matchMedia('(max-width: 767px)').matches) {
+    const element1 = document.querySelector('.speakers__list__container:nth-child(3)');
+    const element2 = document.querySelector('.speakers__list__container:nth-child(4)');
+    const element3 = document.querySelector('.speakers__list__container:nth-child(5)');
+    const element4 = document.querySelector('.speakers__list__container:nth-child(6)');
+
+    const elements = [
+      element1,
+      element2,
+      element3,
+      element4,
+    ];
+
+    elements.forEach((e) => {
+      e.style.display = 'none';
+    });
+
+    speakerButton.addEventListener('click', () => {
+      speakerButton.classList.toggle('show');
+      if (speakerButton.classList.contains('show')) {
+        elements.forEach((e) => {
+          e.style.display = 'block';
+        });
+        speakerButton.innerHTML = `
+        LESS
+        <i class="fa-solid fa-chevron-up"></i>
+        `;
+      } else {
+        elements.forEach((e) => {
+          e.style.display = 'none';
+        });
+        speakerButton.innerHTML = `
+        MORE
+        <i class="fa-solid fa-chevron-down"></i>
+        `;
+      }
+    });
+  }
+};
+showMore();
```

Comment:
In this function (`showMore`), the logic for showing/hiding the speakers could be simplified. Instead of manually selecting each speaker element, you could use a loop or query selector to target all relevant elements dynamically. 😄